addons/mail/models/mail_message.py

```diff
@@ -955,7 +955,7 @@ def _notify_moderators(self):
     # MESSAGE READ / FETCH / FAILURE API
     # ------------------------------------------------------
 
-    def _message_format(self, fnames):
+    def _message_format(self, fetch_followers, fnames):
```

Comment:
I wouldn't change the position of `fnames`.
Also probably `fetch_followers` should have a default `False`.

Reply:
Don't forget to change the signature of overrides in other modules.